src/ServerConfigParams.ts

```diff
@@ -1,191 +1,197 @@
-    url: string;
-    port: number;
-    versionRegistryAddress: string;
-    versionRegistryDelayPeriod?: number;
-    relayHubId?: string;
-    relayHubAddress: string;
-    rskNodeUrl: string;
-    workdir: string;
-    checkInterval: number;
-    readyTimeout: number;
-    devMode: boolean;
-    customReplenish: boolean;
-    registrationBlockRate: number;
-    alertedBlockDelay: number;
-    minAlertedDelayMS: number;
-    maxAlertedDelayMS: number;
-    trustedVerifiers: string[];
-    gasPriceFactor: number;
-    logLevel: LogLevelNumbers;
-    deployVerifierAddress: string;
-    relayVerifierAddress: string;
-    workerMinBalance: number;
-    workerTargetBalance: number;
-    managerMinBalance: number;
-    managerMinStake: string;
-    managerTargetBalance: number;
-    minHubWithdrawalBalance: number;
-    refreshStateTimeoutBlocks: number;
-    pendingTransactionTimeoutBlocks: number;
-    successfulRoundsForReady: number;
-    confirmationsNeeded: number;
-    retryGasPriceFactor: number;
-    maxGasPrice: string;
-    defaultGasLimit: number;
-    estimateGasFactor: number;
-    // revenue-sharing fields
-    // if set to false, the server will reject all the transactions
-    // that require less than the estimated gas.
-    allowForSponsoredTx: boolean;
+  url: string;
+  port: number;
+  versionRegistryAddress: string;
+  versionRegistryDelayPeriod?: number;
+  relayHubId?: string;
+  relayHubAddress: string;
+  rskNodeUrl: string;
+  workdir: string;
+  checkInterval: number;
+  readyTimeout: number;
+  devMode: boolean;
+  customReplenish: boolean;
+  registrationBlockRate: number;
+  alertedBlockDelay: number;
+  minAlertedDelayMS: number;
+  maxAlertedDelayMS: number;
+  trustedVerifiers: string[];
+  gasPriceFactor: number;
+  logLevel: LogLevelNumbers;
+  deployVerifierAddress: string;
+  relayVerifierAddress: string;
+  workerMinBalance: number;
+  workerTargetBalance: number;
+  managerMinBalance: number;
+  managerMinStake: string;
+  managerTargetBalance: number;
+  minHubWithdrawalBalance: number;
+  refreshStateTimeoutBlocks: number;
+  pendingTransactionTimeoutBlocks: number;
+  successfulRoundsForReady: number;
+  confirmationsNeeded: number;
+  retryGasPriceFactor: number;
+  maxGasPrice: string;
+  defaultGasLimit: number;
+  estimateGasFactor: number;
+  /**
+   * (Dis)Allows revenue sharing feature and sets the fee value (%) that the server will take from all transactions.
+   * This fee will be added to the estimated gas and required in the transaction amount.
+   * @option 0 - disables revenue sharing
+   * @option !0 - absolute value of the fee percentage to be added to gas
+   * @note the percentage is represented as a fraction (1 = 100%) string to allow for very low or high percentages
+   * @note the minus sign is omitted if used
+   * @note fractions exceeding the number of decimals of that of the native currency will be rounded up
+   */
+  workerFeePercentage: string;
```

Comment:
If we change the param name, we need to change it here also.

Reply:
done